libs/sourcify/src/client.rs

```diff
@@ -94,12 +92,50 @@ impl Client {
 
         Self::process_sourcify_response(response).await
     }
+
+    pub async fn verify_from_etherscan(
+        &self,
+        chain_id: &str,
+        contract_address: Bytes,
+    ) -> Result<VerifyFromEtherscanResponse, Error<VerifyFromEtherscanError>> {
+        let contract_address = DisplayBytes::from(contract_address);
+        let url = self.generate_url("verify/etherscan");
+
+        #[derive(Serialize)]
+        #[serde(rename_all = "camelCase")]
+        struct Request<'a> {
+            chain_id: &'a str,
+            address: String,
+        }
+
+        let request = Request {
+            chain_id,
+            address: contract_address.to_string(),
+        };
```

Comment:
I think this way is okay, but I just wonder why not simply use serde_json::json
```
let request = serde_json::json!({
    "chainId": chain_id,
    "address": contract_address.
});
```

Reply:
I believe it may done both ways, the separate structure was just the first which came up in my head